testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlterTableCompatibility.java

```diff
@@ -418,7 +419,8 @@ public void testIdentityColumn()
                     .contains("b BIGINT GENERATED ALWAYS AS IDENTITY");
             onDelta().executeQuery("INSERT INTO default." + tableName + " (a) VALUES (0)");
 
-            List<QueryAssert.Row> expected = ImmutableList.of(row(0, 1));
+            DatabricksVersion databricksRuntimeVersion = getDatabricksRuntimeVersion().orElseThrow();
+            QueryAssert.Row expected = databricksRuntimeVersion.isOlderThan(DATABRICKS_122_RUNTIME_VERSION) ? row(0, 1) : row(0, 2);
```

Comment:
add `// Actual value for IDENTITY column varies between Databricks versions`

Is it always `2`? (or flaky test candidate?)

Reply:
It was always 2 as far as I tested, but yes this might be flaky test candidate. Let's tweak the assertion if they return a different value in the future.